packages/core/src/public/images/app/cashier/cashier-demo-dark.svg

```diff
@@ -0,0 +1,114 @@
+                            <g>
+                                <path fill="#85ACB0" d="M24 12c0 6.627-5.373 12-12 12S0 18.627 0 12 5.373 0 12 0s12 5.373 12 12" transform="translate(24 7) translate(0 1)"/>
+                                <path fill="#FFF" fill-rule="nonzero" d="M13.527 4C17.588 4 21 7.796 21 12c0 4.204-3.412 8-7.473 8H7c-.552 0-1-.448-1-1s.448-1 1-1h1v-3H7c-.513 0-.936-.386-.993-.883L6 14c0-.552.448-1 1-1h1v-2H7c-.513 0-.936-.386-.993-.883L6 10c0-.552.448-1 1-1h1V6H7c-.513 0-.936-.386-.993-.883L6 5c0-.552.448-1 1-1zm0 2H10v3h3c.513 0 .936.386.993.883L14 10c0 .552-.448 1-1 1h-3v2h3c.513 0 .936.386.993.883L14 14c0 .552-.448 1-1 1h-3v3h3.527c2.839 0 5.353-2.725 5.469-5.781L19 12c0-3.144-2.567-6-5.473-6z" transform="translate(24 7) translate(0 1)"/>
+                            </g>
+                        </g>
+                    </g>
+                </g>
+            </g>
+        </g>
+        <g fill-rule="nonzero" transform="translate(29 35)">
+            <path fill="#DC4650" d="M69.994 73.475c1.645-1.302 4.033-1.03 5.341.61l22.407 28.078c1.302 1.632 1.035 4.01-.597 5.312l-.011.009c-1.645 1.302-4.033 1.029-5.341-.61L69.386 78.794c-1.302-1.631-1.035-4.01.597-5.311l.011-.01z"/>
+            <path fill="#85ACB0" d="M75.514 79.975c3.189-2.522 7.816-1.99 10.35 1.188l32.971 41.361c2.522 3.163 2.002 7.771-1.161 10.293l-.022.017c-3.189 2.522-7.816 1.99-10.35-1.188L74.33 90.286c-2.521-3.164-2.001-7.772 1.162-10.293l.022-.018z"/>
+            <path fill="#FFF" fill-opacity=".291" d="M75.961 86.815c.633-.5 1.552-.395 2.055.236l32.952 41.337c.5.628.397 1.543-.231 2.043l-.004.003c-.633.501-1.552.396-2.055-.235L75.726 88.862c-.5-.628-.397-1.543.23-2.043l.005-.004z"/>
+            <path fill="#DC4650" d="M44 0C19.7 0 0 19.476 0 43.5S19.7 87 44 87s44-19.476 44-43.5C87.996 19.477 68.299.004 44 0zm0 76.818c-18.613 0-33.701-14.917-33.701-33.318 0-18.401 15.088-33.318 33.701-33.318S77.701 25.099 77.701 43.5c0 18.401-15.088 33.318-33.701 33.318z"/>
+            <path fill="#151717" stroke="#151717" stroke-width="10" d="M44.5 10C26 10 11.003 24.996 11 43.495c-.003 18.5 14.991 33.5 33.49 33.505 18.5.005 33.502-14.986 33.51-33.486.005-8.888-3.522-17.413-9.805-23.7C61.912 13.53 53.388 9.999 44.5 10z"/>
+            <use fill="#000" xlink:href="#prefix__f"/>
+            <use fill="#FFF" xlink:href="#prefix__f"/>
+        </g>
+    </g>
+</svg>
```

Comment:
missing svgo

Reply:
@negarn isn't svgo a webpack loader performing its task on build time?! Do I have to use is manually somewhere?

Reply:
@mahdiaryayi-fs if you check our other .svg files they should all be one liners because we run svgo on them so we can store them as minimally as possible

Reply:
Done!